### PR TITLE
community/pdns-recursor: security upgrade to 4.1.8

### PR DESCRIPTION
**Pull request by J0WI**
https://blog.powerdns.com/2018/11/09/powerdns-recursor-4-1-7-released/
See also #5592

### CODE REVIEW

File: community/pdns-recursor/APKBUILD [hunk 1]
```diff
@@ -24,6 +23,9 @@ options="!check"
 builddir="$srcdir/$pkgname-$pkgver"
 
 # secfixes:
+#   4.1.7-r0:
+#     - CVE-2018-10851
+#     - CVE-2018-14626
```
Comment (tcely) [inline community/pdns-recursor/APKBUILD]: Why was `CVE-2018-14644` not listed here?

**Comment by J0WI** [comment]
CVE-2018-14644 has been fixed in 4.1.4. I don't know why the `secfixes` comment hasn't been added in https://github.com/alpinelinux/aports/commit/009fc7167b17f8d97f9c89828d81b905d947b4aa.

**Comment by tcely** [comment]
@J0WI `4.1.4` did not contain a fix for `CVE-2018-14644` so it wouldn't be included in that commit.

https://docs.powerdns.com/recursor/security-advisories/powerdns-advisory-2018-07.html

**Comment by J0WI** [comment]
Thanks for clarify, I've updated the PRs.

**Comment by J0WI** [comment]
@tcely is there anything left to get these security updates merged?

**Comment by J0WI** [comment]
Fixed in 0756f3f95a7b96bfe212a78d42d64da35b03a78a and 8c3bc201054c136b08669104b22bf6105dee54de

**Comment by J0WI** [comment]
This has been fixed in 0756f3f95a7b96bfe212a78d42d64da35b03a78a and 8c3bc201054c136b08669104b22bf6105dee54de

**Pull request closed without merging**